src/utils/file-upload-utils.js

```diff
@@ -47,7 +47,7 @@ const scanFileForVirus = (fileBuffer, timeout) => {
     apiInstance.scanFile(fileBuffer, (error, data) => {
       if (error) {
         logger.error(
-          `Error when calling Cloudmersive Virus Scan API: ${error.message}`
+          `Error when calling Cloudmersive Virus Scan API for the following file content: ${fileBuffer}: ${error.message}`
```

Comment:
does this show the file extension?

fileBuffer is not the binary right? are we printing out the entire media inside the buffer?

Reply:
I was thinking we needed the actual file content to debug issues with cloudmersive - what we're logging here is exactly what we submit to cloudmersive so that we can debug if necessary

Reply:
Hmm would that be helpful? a file buffer is going to be huge, what we probably need is the actual file extension and the mime type - https://www.npmjs.com/package/mime-types?